packages/core/src/App/app.jsx

```diff
@@ -55,6 +55,7 @@ const App = ({ root_store }) => {
         setUrlLanguage(getLanguage());
         initFormErrorMessages(FORM_ERROR_MESSAGES);
         setSharedMT5Text(MT5_TEXT);
+        handleResize();
```

Comment:
Is this needed? the `function` is already added to the event listener.

Reply:
When we don't call it here, it does not show the message in landscape mode if the page is loading in landscap mode from the beginning.